src/components/entries/NumberField.js

```diff
@@ -103,25 +113,66 @@ export default function NumberFieldEntry(props) {
     setValue,
     step,
     onFocus,
-    onBlur
+    onBlur,
+    validate
   } = props;
 
-  const value = getValue(element);
+  const [ cachedInvalidValue, setCachedInvalidValue ] = useState(null);
+  const globalError = useError(id);
```

Comment:
This also enables setting the errors from outside via linting, which is nice given we might want to enable it for quite some number properties in forms 👍